packages/widgets/src/blocks/legacy-widget/editor.scss

```diff
@@ -114,10 +114,15 @@
 	width: 100%;
 }
 
-.wp-block-legacy-widget__edit-no-preview {
+.wp-block-legacy-widget__edit-no-preview.wp-block-legacy-widget__edit-no-preview.wp-block-legacy-widget__edit-no-preview {
```

Comment:
I have seen this pattern more and more often :) Repeating classname syndrome. I guess it's the only way? (At least this is the constant answer).

Reply:
I don't like it either, but unfortunately it seems so – it's just the charm of applying theme styles in an environment where we have a bunch of other stylesheets already. Maybe some planning and a major refactor of how we work with the styles would spare us doing this in the future (maybe), but I can't see a simple solution.